### PR TITLE
Improve some setting descriptions in the graphics options.

### PR DESCRIPTION
**Pull request by pauldacheez**
More than a few of these have annoyed me for a while, what with their typos and odd wording and occasional lack of important details.

Criticisms welcome. I mean, as long as it doesn't devolve into MORE bikeshedding. Even I don't enjoy that.

### CODE REVIEW

**Comment by robertybob** [comment]
@delroth @Sonicadvance1 Are any amendments needed or is this ready to be merged?

**Comment by Buddybenj** [comment]
I still think my last comment should be actioned because the sentence sounds weird (and I think a rebase would be worth it), however it's only a minor gripe. If others think its fine as is, I'm all for this being merged.

File: Source/Core/DolphinWX/VideoConfigDiag.cpp [hunk 1]
```diff
@@ -98,11 +98,11 @@ void VideoConfigDiag::Event_Close(wxCloseEvent& ev)
 }
 
 #if defined(_WIN32)
-static wxString backend_desc = wxTRANSLATE("Selects what graphics API to use internally.\nThe software renderer is only used for debugging, so you'll want to use either Direct3D or OpenGL. Different games will behave differently on each backend, so for best emulation experience it's recommended to try both and chose the one that fits your requirements best.\nNote that the Direct3D backend is not available on old Windows versions.\n\nIf unsure, use OpenGL.");
+static wxString backend_desc = wxTRANSLATE("Selects what graphics API to use internally.\nThe software renderer is extremely slow and only useful for debugging, so you'll want to use either Direct3D or OpenGL. Different games will behave differently on each backend, so for the best emulation experience it's recommended to try both and choose the one that has less issues.\n\nIf unsure, use OpenGL.");
```
Comment (Buddybenj) [inline Source/Core/DolphinWX/VideoConfigDiag.cpp]: I don't understand. The page you linked to says to use less with uncountable things and units of measurement and fewer everywhere else. This isn't uncountable nor is it a unit of measurement, so according to that page fewer should be used, but I'm fine with using less if you want.

**Comment by Sonicadvance1** [comment]
@dolphin-emu-bot rebuild

**Pull request merged**